server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -62,6 +62,12 @@ grant codeBase "${codebase.elasticsearch-core}" {
   permission java.lang.RuntimePermission "getClassLoader";
 };
 
+//grant codeBase "${codebase.elasticsearch-nalbind}" {
+grant {
```

Comment:
TODO

Reply:
This might be completely unnecessary now, especially with the list proxies removed.